examples/ant/compilerclasspath/build.xml

```diff
@@ -0,0 +1,35 @@
+<!--
+  Copyright 2015 Google Inc. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  -->
+<project name="error prone example" default="compile">
+  <path id="errorprone">
+    <pathelement location="../../../ant/target/error_prone_ant-2.0.2-SNAPSHOT.jar"/>
```

Comment:
It would be nice if the version number were not hardcoded here.  Any ideas?  At worst we should at least document updating this in our release instructions.

Also do we really want this pointing to the SNAPSHOT jar?